packages/proxy/lib/http/index.ts

```diff
@@ -207,6 +214,7 @@ export function _runStage (type: HttpStages, ctx: any, onError: Function) {
       try {
         middleware.call(fullCtx)
       } catch (err) {
+        errorUtils.logError(err)
```

Comment:
`onError` already logs the internal error via `debug`, do we need this additional log here?

Side note... errorUtils.logError doesn't log error.cause, whereas `util.inspect` does. If you keep this and add the `cause`, it would be worth adding `cause` logging to `logError`. Probably cuz `cause` wasn't supported until Node `16.9.0`

Reply:
The actual double-call to `this.next` behavior has been cleaned up since 10.3.0 so, right now, an error like the one from the original issue is pretty much swallowed from the user's perspective - the request fails but there's no logging to the terminal at all to indicate the issue. The error is present if you inspect the failed network call from the command log or enable debug logging, but nobody thought to do that in the original issue so it's apparently not an intuitive thing to look at.

I'd like to find a way to hint to the user that they should dig deeper to find the real issue (either inspect the failed request, or enable debug logging), but we don't really have a pattern for that. Dumping the entire error into the log is definitely not ideal though, so I'd prefer to find something with a lighter touch. Dunno, maybe I'm over-thinking it

Reply:
Fair enough, I see how this simplifies reporting proxy errors for sure, which seems like a good thing. In that case, can you augment the log with some info about the request receiving the error? Otherwise there's no context for the log. Something like:

```suggestion
	err.message = `Internal error while proxying "${ctx.req.method} ${ctx.req.proxiedUrl}" in ${middlewareName}:\n${err.message}`
        errorUtils.logError(err)
```